packages/@uppy/webcam/tsconfig.build.json

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": "../../../tsconfig.shared",
+  "compilerOptions": {
+    "noImplicitAny": false,
+    "outDir": "./lib",
+    "paths": {
+      "@uppy/utils/lib/*": ["../utils/src/*"],
```

Comment:
Why not just:

```suggestion
      "@uppy/utils/*": ["../utils/src/*"],
```

Reply:
Because that would be incorrect? `@uppy/utils/lib/whatever` should link to `@uppy/utils/src/whatever.ts`, not `@uppy/utils/src/lib/whatever`

Reply:
It's a path alias, so it doesn't matter what you name it, you could name it `🤡` and it would still work.

Reply:
Of course it matters, we want the output file to match the correct path – or we'd need some other tool to transform `🤡` to the correct specifier.

Reply:
Your build tool doesn't transform path aliases? Fair enough, then it matters 😄